CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## Unreleased
 
-...
+Enhancements:
+- Enable explicitly specified null values to satisfy System.Nullable<> dependencies (@jnm2, #521)
```

Comment:
Would you consider this an enhancement, and should I add any more detail?

Reply:
Enhancement sounds good since this wasn't a supported feature.

That's enough detail. Usually if you think there is more detailed required then I usually push people to add that to the docs rather than the changelog.